src/Aspire.Hosting.Azure.Functions/AzureFunctionsProjectResourceExtensions.cs

```diff
@@ -100,6 +100,8 @@ public static class AzureFunctionsProjectResourceExtensions
                 context.EnvironmentVariables["OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY"] = "in_memory";
                 context.EnvironmentVariables["ASPNETCORE_FORWARDEDHEADERS_ENABLED"] = "true";
                 context.EnvironmentVariables["FUNCTIONS_WORKER_RUNTIME"] = "dotnet-isolated";
+                // Required to enable OpenTelemetry in the Azure Functions host.
+                context.EnvironmentVariables["AzureFunctionsJobHost__telemetryMode"] = "OpenTelemetry";
```

Comment:
@fabiocav - why does this not follow the same naming pattern as `FUNCTIONS_WORKER_RUNTIME`? Different prefix, double underscores, etc.

Reply:
Is this the exact casing of the environment variable? I believe that linux environment variables are case sensitive.

Reply:
IConfiguration

Reply:
> IConfiguration

Yes. The other variables are explicitly consumed as environment variables by the host, while the new variable being added overrides the configuration typically defined in `host.config`, which is pulled into configuration and mapped to a section named `AzureFunctionsJobHost`.